src/infra/logging/logger.ts

```diff
@@ -0,0 +1,4 @@
+import { ConsoleLogger, Injectable, Scope } from '@nestjs/common';
+
+@Injectable({ scope: Scope.TRANSIENT })
+export class Logger extends ConsoleLogger {}
```

Comment:
In mehreren Files fehlt die leere Zeile am Ende. Ich schätze mal, du hast prettier nicht so eingerichtet, dass es beim Speichern formatiert.